spec/defines/vhost_spec.rb

```diff
@@ -7,7 +7,11 @@
         facts
       end
 
-      let(:pre_condition) { 'include caddy' }
```

Comment:
No need for a pre_condition as the the caddy class is inow ncluded in the define.